modules/network/fantom.ts

```diff
@@ -246,6 +246,14 @@ const fantomNetworkData: NetworkData = {
                 isIbYield: true,
             },
         },
+        defaultHandlers: {
+            ankrETH: {
+                tokenAddress: '0x12d8ce035c5de3ce39b1fdd4c1d5a745eaba3b8c',
+                sourceUrl: 'https://api.staking.ankr.com/v1alpha/metrics',
+                path: 'services.{serviceName == "eth"}.apy',
+                isIbYield: true,
+            },
+        },
```

Comment:
this is already added via the ankr YB apr service. remove the ankr specific service if it works with the default one. For all chains

Reply:
good point, missed that